catacombs-chimeras-service/src/main/java/cz/muni/fi/service/TroopServiceImpl.java

```diff
@@ -143,6 +146,21 @@ public TroopsAvgExpReportDTO countTroopsAverageExperienceReport() {
                 (listOfReportItems.isEmpty()) ? 0 : listOfReportItems.get(listOfReportItems.size() - 1).getAverage(),
                 (listOfReportItems.isEmpty()) ? 0 : listOfReportItems.get(0).getAverage());
     }
+    
+    @Override
+    public TroopWealthDTO getMoneyPerHeroList() {
+        List<TroopWealthItemDTO> troops = new ArrayList<>();
+        for(Troop troop : findAllTroops()) {
+            if(troop.getHeroes().isEmpty() || troop.getAmountOfMoney() == null) {
```

Comment:
cs, kontrolovat `getAmountOfMoney` je imo zbytečné (entita Troop nemůže mít nově amount of money `null`